### PR TITLE
Remove ENV_PATH on Black action completion.

### PR DESCRIPTION
**Pull request by cjproud**
### Description

Relates to #3708 by deleting the `.black.env` folder specified by `ENV_PATH` on action completion.

I had a difficult time replicating the issue above @JelleZijlstra and I can't see that any changes were made to stable that would effect this behaviour. Probably worth deleting that `.black.env` dir anyway?

### Checklist - did you ...

- [X] Add an entry in `CHANGES.md` if necessary?
- [X] Add / update tests if necessary?
- [X] Add new / update outdated documentation?

### CODE REVIEW

File: action/main.py [hunk 1]
```diff
@@ -73,5 +74,6 @@
         stderr=STDOUT,
         encoding="utf-8",
     )
+shutil.rmtree(ENV_PATH)
```
Comment (ichard26) [inline action/main.py:77]: Is it worth passing `ignore_errors=True` to this call? An error is unlikely, but it also shouldn't fail the entire action IMO.

**Review by ichard26 — COMMENTED** [review]
Yeah, I'm not sure how this would cause issues, but this is a good idea either way. Thanks for filing the PR!

Please add a changelog entry though.

**Comment by ichard26** [comment]
Don't mind the PyPy failures, it seems like a recent PyYAML release broke things on PyPy... :/

File: action/main.py (hunk 1 above)
Comment (cjproud) [reply action/main.py:77]: I agree, I think an error is unlikely but yeah shouldn't fail the entire action.

**Comment by cjproud** [comment]
@ichard26 I've added a changelog entry and the `ignore_errors=True` flag to the `shutil.rmtree` function 👍

**Review by hauntsaninja — APPROVED** [review]
Thanks, looks like another user confirmed the fix as well!

**Pull request merged**